packages/apps/asana/src/services/asana.ts

```diff
@@ -57,61 +57,61 @@ export default class AsanaService extends TransactionBaseService {
 
     let documents: IndexableDocument[] = [];
     try {
-      const projects = await this.getAsanaProjects(oauth.token, last_sync);
-      for (const project of projects) {
-        const tasks = await this.getAsanaTasks(
-          oauth.token,
-          project.gid,
-          last_sync
-        );
-        for (const task of tasks) {
-          const doc: IndexableDocument = {
-            id: task.gid,
-            organisationId: org.id,
-            title: task.name,
-            source: AppNameDefinitions.ASANA,
-            sections: [
-              {
-                link: `https://app.asana.com/0/${project.gid}/${task.gid}`,
-                content: task.notes,
-              },
-            ],
-            type: DocType.TXT,
-            updatedAt: new Date(task.modified_at),
-            metadata: {
-              completed: task.completed,
-              project_id: project.gid,
-              project_name: project.name,
-              project_link: `https://app.asana.com/api/1.0/projects/${project.gid}`,
-            },
-          };
-          documents.push(doc);
-          if (documents.length >= 100) {
-            yield documents;
-            documents = [];
-          }
-        }
-      }
-      yield documents;
+      // const projects = await this.getAsanaProjects(oauth.token, last_sync);
```

Comment:
Can you explain why these lines are commented out?

Reply:
Because to demonstrate the API token demo , as we were passing wrong API token , the server was crashing , hence commented the  code

Reply:
Same Comment Here!! The moment you submit this Asana will break.

Reply:
Ok , I understand that unless until the new strategy is implemented , I should return and not start the oculation process , is this what you mean such that , it will not break stuff as you mentioned earlier !

`if(new_strategy){
         return;
}`